docs/src/main/tut/cors.md

```diff
@@ -95,12 +95,12 @@ corsMethodSvc(duckPost).unsafeRun
 
 As you can see, the CORS headers were only added to the `GET` and `POST` requests.
 Next, we'll create a configuration that limits the origins to "yahoo.com" and
-"duckduckgo.com".
+"duckduckgo.com". allowedOrigins can use any expression that resolves into a boolean.
 
 ```tut:book
 val originConfig = CORSConfig(
   anyOrigin = false,
-  allowedOrigins = Some(Set("yahoo.com", "duckduckgo.com")),
+  allowedOrigins = origin => if(Set("yahoo.com", "duckduckgo.com").contains(origin)) true else false,
```

Comment:
`if (x) true else false` can be simplified to `x`.

`Set[String]` is a `String => Boolean`, so it could even just be `Set(...)`, but this might be less obvious.